verify-action.md

```diff
@@ -0,0 +1,123 @@
+# Verify image with Notation in GitHub Actions
+
+This document walks you through how to create a GitHub Actions workflow using Notation to verify a signed image stored in Azure Container Registry (ACR).
+
+## Prerequisites
+
+- You have a signed image stored in your ACR. If not, you can follow the document [Sign an image with Notation in GitHub Actions](sign-action.md) to create one.
```

Comment:
I forgot this in the last review. If the user already has an image stored in their ACR, they don't need to follow the `sign` document. In this case, shall we mention that they still need to save a `Repository secret` named `ACR_PASSWORD` in their github repo?